src/main/scala/strawman/collection/Factories.scala

```diff
@@ -26,6 +28,10 @@ trait IterableFactory[+CC[_]] {
 
 }
 
+trait IterableFactoryWithBuilder[+CC[_]] extends IterableFactory[CC] {
+  def newBuilder[A](): Builder[A, CC[A]]
+}
```

Comment:
As explained in the document at the bottom of the diff, it would be useful to have more power and also add the following method:

~~~ scala
implicit def canBuild[A]: CanBuild[A, CC[A]] = () => newBuilder[A]
~~~

Where `CanBuild` is the type defined in #80. If we add this line and if we also add `XxxWithBuilder` factories for the other kind of factories (`SortedIterable`, `Map` and `SortedMap`), we would be able to implement the tests given in #80.